src/util.ts

```diff
@@ -22,7 +22,7 @@ const toBool = (value: unknown, defaultValue?: boolean) => {
     }
   }
 
-  if (typeof value === "undefined") {
+  if (typeof value === "undefined" || value === "") {
```

Comment:
Would it make more sense to always just `return Boolean(defaultValue);` and not throw an error?

Reply:
Yeah, I like that!